src/navigate-pop/navigate-pop.md

```diff
@@ -0,0 +1,15 @@
+<!-- reference: 
+everything: https://support.system76.com/articles/pop-basics
+cosmic: https://github.com/pop-os/cosmic
+pop shell: https://github.com/pop-os/shell
+ubuntu vs. pop: https://support.system76.com/articles/difference-between-pop-ubuntu/
+-->
+
+# Navigate Pop!\_OS
+
+The Pop!\_OS Desktop environment is based on the GNOME shell and layered with other components called Pop!_Shell and C.O.S.M.I.C. (Computer Operating System Main Interface Components).
+
+- [Manipulate windows using Tiling and Stacking.](tiling-stacking-windows.md)
```

Comment:
If you're going with `Auto Tiling` as the proper noun name of the feature on the tiling page, I would use `Auto Tiling` here. Otherwise, de-capitalize these words so it's just `using tiling and stacking`.